### DISCUSSION TITLE
Automatic citation  numbering when rendering a quarto document to word

### DISCUSSION THREAD

**Discussion by charldelaoa** [quarto-dev/quarto-cli#6131] [Q&A] [answered]
### Description

Hello,

I am working with a quarto book rendering rendering .qmd files into word. 

I have a .bib reference file and I'm adding citations using: "@citation1". When rendering to html this adds a hyperlink to the refererence, however when rendering to word the citation is only plain text, e.g.: "Knuth (1984)" - there is no citation or numbering. A reference list is generated at the end.

Is there a way to add automatic numbering and hyperlinks to the references to the citations in word, similar as when generating a pdf from a latex document?

Carlos

**Comment by charldelaoa** [comment]
I've also noticed that when rendering to pdf there is not an automatic numbering/hyperlink generated. I have written documents in latex and this is done automatically, which I think is very powerful. It would be nice to have option for quarto documents specially for word/pdf documents as most people I collaborate with are used to those formats...

**Comment by mcanouil** [comment]
Even if Word is supported, its support by Pandoc, thus Quarto is not the best.

---

Could you share a small self-contained "working" (reproducible) example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````

**Reply by cderv** [reply]
> Even if Word is supported, its support by Pandoc, 

To add to that, note that Docx native citation is still a feature request in Pandoc 
* https://github.com/jgm/pandoc/issues/7889

**Comment by cderv** [comment, marked as answer]
Did you specify `link-citations: true` ?  See: https://quarto.org/docs/reference/formats/docx.html#references 

> link-citations
> If true, citations will be hyperlinked to the corresponding bibliography entries (for author-date and numerical styles only). Defaults to false.

**Comment by charldelaoa** [comment]
Thank you, this last suggestion adds hyperlinks in word.
I was wondering if automatic numbering can be generated instead of the default, e.g.:

If I have a book reference:

@book{test,
title={Deep learning},
author={author1 and author2 and author3},
year={2016},
publisher={a}
}

When adding "See @test " to the .qmd file the output will be:

See author1, author2, and author3 (2016) - hyperlink under 2016,
Would it be possible to have see [1] where 1 is the reference number instead?

**Reply by cderv** [reply]
This is probably something handle by citation style using CSL. Did you already looked at that ? https://quarto.org/docs/authoring/footnotes-and-citations.html#citation-style
